src/main/webapp/i18n/en/global.json

```diff
@@ -299,5 +299,6 @@
         "connected": "Connected",
         "disconnected": "Disconnected"
     },
-    "loading": "Loading..."
+    "loading": "Loading...",
+    "completed": "Completed"
```

Comment:
This appears to be missing in the `de` file.

Reply:
Good catch! 9ca89cdfced9af0b6b1f3e7cfd1a7b842ce9634a